### ISSUE TITLE
Make official releases

### ISSUE THREAD

**Issue by SuperQ** [google/mtail#41]
Hey, it would be nice to have official releases of mtail when things are in good feature release points.

Also add travis/circleci type auto-builds as well, as we do for various Prometheus components.

**Comment by jaqx0r** [comment]
It would be. I still have 3 features, I think, before we get to a release I
will guarantee with a version tho.

**Comment by jaqx0r** [comment]
Most notably this is blocked on #24, which is in progress.

**Comment by SuperQ** [comment]
Let me know if there's anything I can do to help with #24.  I just yesterday had to make an ugly hack to to deal with some floats[0]. :-)

[0]: https://gitlab.com/gitlab-cookbooks/gitlab-prometheus/merge_requests/183/diffs#7f1349cd98fdf11a95256ee5aaa4cddba801454c_0_32

**Comment by opsnull** [comment]
any update?

**Comment by jaqx0r** [comment]
As long as travis says it's green, it's Production Ready!

With the recent floating point metrics, I expect there to be more teething issues.  I'm not sure I want to commit to an Official Release when the language might still change.

Also, the bugs #61 and #66 concern me enough that I'd rather fix them than tag a version.

**Comment by jaqx0r** [comment]
OK I caved and made a v3.0.0-rc0 tag just now.

**Comment by Lusitaniae** [comment]
@jaqx0r, regarding the 2nd point... Can we include the binaries in upcoming Github releases?

Maybe I can make  PR? 
We can consider using promu + CircleCi like typical prometheus exporters.
